src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/StrandBiasTest.java

```diff
@@ -21,6 +21,7 @@
 
     protected static final int ARRAY_DIM = 2;
     protected static final int ARRAY_SIZE = ARRAY_DIM * ARRAY_DIM;
+    public static final String DISCARDED_MATE_READ_TAG = "DM";
```

Comment:
This constant belongs in the class that *applies* the tag.

Reply:
done